webroot/js/elements/login-dialog.ctrl.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('app')
+        .controller('LoginDialogController', ['$mdDialog', function($mdDialog) {
+            var vm = this;
+
+            vm.showDialog = showDialog;
+            
+            ///////////////////////////////////////////////////////////////////////////
+
+            function showDialog(url) {
+                $mdDialog.show({
+                    controller: DialogController,
+                    templateUrl: get_tatoeba_root_url() + '/users/login_dialog_template?redirect=' + url,
```

Comment:
The redirection URL is not escaped. This is a problem if the URL contains `&`.

For example, if I open https://dev.tatoeba.org/eng/sentences_lists/index?sort=name&direction=asc as a guest, and login from there, I am redirected to https://dev.tatoeba.org/eng/sentences_lists/index?sort=name (without &direction=asc).